src/colorizer/CanvasOverlay.ts

```diff
@@ -242,7 +256,11 @@ export default class CanvasOverlay implements IRenderCanvas {
   // TODO: Move `isExporting` flag into state
   public setIsExporting(isExporting: boolean): void {
     this.isExporting = isExporting;
-    this.render(false);
+    this.render({ renderInnerCanvas: false });
```

Comment:
I also updated `render()` to take an options object too because I found that just the plain boolean argument wasn't very intuitive (and it actually means something different in the inner canvases)!